index.md

```diff
@@ -178,6 +178,17 @@ instructions:
     - title: Community Supported
       summary: Packaged and supported by the wider community.
       distros:
+        - name: Alpine Linux
+          id: alpine
+          instructions: >-
+            Packages are available from the [community respositories](https://pkgs.alpinelinux.org/packages?name=openrazer).
+
+
+                doas apk add openrazer
+
+
+            The kernel modules are built via akms.
```

Comment:
Maybe link to upstream reference to akms since probably no-one reading this will know akms.

Reply:
Done